browser_patches/firefox-beta/juggler/screencast/ScreencastEncoder.cpp

```diff
@@ -113,7 +113,11 @@ class ScreencastEncoder::VPXFrame {
     VPXFrame(rtc::scoped_refptr<webrtc::VideoFrameBuffer>&& buffer, const gfx::IntMargin& margin)
         : m_frameBuffer(std::move(buffer))
         , m_margin(margin)
-    { }
+    {
+      // YUV offsets must be even.
+      m_yuvTopOffset = m_margin.top & 1 ? m_margin.top + 1 : m_margin.top;
```

Comment:
nit: I'd use  `m_yuvTopOffset = m_margin & ~1;` and moved this inside `convertToVpxImage` where we compute all other yuv related stuff.

Reply:
I kept rounding to top so that it doesn't cut the top-left corner. Other then that, inlined it in